content/docs/tasks/traffic-management/tcp-traffic-shifting/index.md

```diff
@@ -0,0 +1,172 @@
+---
+title: TCP Traffic Shifting
+description: Shows you how to migrate TCP traffic from an old to new version of a TCP service.
+weight: 25
```

Comment:
change the weight to 26 so it comes after the http traffic shifting task

Reply:
Addressed in 2909c94.